src/providers/sh/commands/login.js

```diff
@@ -110,6 +110,14 @@ const getVerificationData = async ({ apiUrl, email }) => {
     )
   }
 
+  if (!res.ok) {
+    debug('error response from POST /now/registration: %d %j', res.status, body)
+    const message = body.error && body.error.code === 'invalid_email'
+        ? 'Invalid email address'
+        : 'Unexpected error. Please try again later'
```

Comment:
I think we should retain:

`Unexpected error: ${err.message}`

Reply:
fixed